### ISSUE TITLE
I can't believe, you forced us to implement this, and yet people can buy our NFTs with 0% fees on X2Y2

### ISSUE THREAD

**Issue by DaWe35** [ProjectOpenSea/operator-filter-registry#83]
We almost rescheduled the mint, because without `[operator-filter-registry](https://github.com/ProjectOpenSea/operator-filter-registry)`, OpenSea won't enforce creator fees after January 2. So we were kinda forced to implement this, otherwise no one will pay fees.

Then as it turned out, it meant nothing, people can still buy/sell on X2Y2 with 0% creator fees. When I think it can't be worse, OpenSea proves there is always worse, and you charge you a ridiculous 2.5% fee for providing shit infrastructure. And if you want access to the API, fill a fckn google form and pray... WTF is wrong with the OpenSea management??? Ohh, I know you love money 💵💵💵💵 and take a shit about everyone

**Comment by cagataygulten** [comment]
Hey, What if you claim that collection on X2Y2 too and put a royalty fee?

**Comment by DaWe35** [comment]
@cagataygulten then it works. My problem is that this repo give the false intention of forcing creator fees, while in reality you need to do the setup on other exchanges too. What's the point of storing the creator fee on-chain then?

**Comment by cagataygulten** [comment]
Yeah that is true, This package has no effects to enforce creator fee on-chain and you need to set on each marketplace seperately. As I understand, this package only filters marketplaces you can not set a creator fee. Here is the list of these: https://github.com/ProjectOpenSea/operator-filter-registry#filtered-addresses. 

So you will have a enforced creator fee if you set it on all other marketplaces. It is still up to you.

**Comment by DaWe35** [comment]
@cagataygulten I can set the fee on LooksRare, then why it is included?
Also, why don't they write about that? If this repo is called as something what enforces fees, I'd expect to do that.
